packages/modular-scripts/src/utils/getPackageDependencies.ts

```diff
@@ -55,30 +56,67 @@ export async function getPackageDependencies(
     path.join(targetLocation, 'package.json'),
   ) as CoreProperties;
 
+  const lockFile = fs.readFileSync(path.join(getModularRoot(), 'yarn.lock'), {
+    encoding: 'utf8',
+    flag: 'r',
+  });
+
   const deps = Object.assign(
     Object.create(null),
-    targetManifest.devDependencies,
     rootManifest.devDependencies,
-    targetManifest.dependencies,
     rootManifest.dependencies,
+    targetManifest.devDependencies,
+    targetManifest.dependencies,
   ) as Dependency;
 
-  /* Get regular dependencies from package.json (regular) or root package.json (hoisted)
-   * Exclude workspace dependencies. Error if a dependency is imported in the source code
+  const lockDeps = await buildDepTree(
+    // Build a dependency tree from the lockfile, using target dependencies and root dependencies in order of specificity
+    JSON.stringify(
+      Object.assign(Object.create(null), targetManifest, {
```

Comment:
This is following a comment by @steveukx - there might be dependencies called as object default members.

Reply:
This one would be fine as a `{ ...spread }` - it wasn't in the other case because the resulting object was used with a `if (key in object)` check without an `object.hasOwnProperty(key)` filter.

Reply:
This one would be fine as a `{ ...spread }` - it wasn't in the other case because the resulting object was used with a `if (key in object)` check without an `object.hasOwnProperty(key)` filter.